ipfs-pinning-service.yaml

```
@@ -440,7 +440,7 @@ components:
       uniqueItems: true
       minItems: 0
       maxItems: 20
-      example: ['/p2p/QmSourcePeerId']
+      example: ['/ip4/1.2.3.4/tcp/4001/p2p/QmSourcePeerId','/ip4/5.6.7.8/udp/4001/quic/p2p/QmSourcePeerId','/dnsaddr/ipfs-nodes.example.com']
```

This looks good, as @obo20 suggests I think it would be good to include in the description that the PeerId must be included in the multiaddr.
```suggestion
      example: ['/ip4/1.2.3.4/tcp/4001/p2p/QmSourcePeerId','/ip4/5.6.7.8/udp/4001/quic/p2p/QmSourcePeerId','/dnsaddr/ipfs-nodes.example.com/p2p/QmSourcePeerId']
```

---

While not mandatory for `/dnslink` in this context (we don't care about connecting to a specific PeerID), I agree, safer to include it so people  don't omit it in other contexts.

---

Hm.. actually, its safer to remove `/dnsaddr` from this example.
I think if used, it would be in `PinStatus.delegates` instead.

---

Fixed in [`7802641` (#70)](https://github.com/ipfs/pinning-services-api-spec/pull/70/commits/78026413d14dee2a4796e858457f61e1a679dd7d)